applitools.config.js

```diff
@@ -1,7 +1,7 @@
 const { execSync } = require('child_process');
 const appName = require('./package.json').name;
 const merge = require('lodash/merge');
-const { DATA_READY_HOOK } = require('./test/visual/dataHooks.js');
+const { DATA_READY_HOOK } = require('storybook-snapper/dist/src/index');
```

Comment:
I'd ditch this one as discussed privately

Reply:
So you're saying `storybook-snapper` shouldn't have a default selector for this?

My only problem is that we need this selector both in the config file, and in the storybook  react component.
I think their design is kind of limiting in this way.

And since I really like having a single source of truth, I'll still need to import the hook from a datahook file.